router.go

```diff
@@ -59,8 +59,8 @@ func (r *Router) Add(method, path string, h HandlerFunc) {
 	if path[0] != '/' {
 		path = "/" + path
 	}
-	ppath := path        // Pristine path
-	pnames := []string{} // Param names
+	ppath := path       // Pristine path
+	var pNames []string // Param names
```

Comment:
Please revert this and rename to `pnames`.

Reply:
fixed